src/amicableNumbers/AmicableNums.java

```diff
@@ -0,0 +1,85 @@
+package amicableNumbers;
+
+import java.util.ArrayList;
+
+public class AmicableNums {
+
+	private static final int LIMIT = 1000000;
+	private static ArrayList<Integer> checked1;
+	private static ArrayList<Integer> checked2;
+	private static long s1, f1, s2, f2;
+	
+	public static void main(String[] args) {
+		
+		/*
+		s1 = System.currentTimeMillis();
+		checked1 = new ArrayList<Integer>();
+		
+		for(int x = 1; x <= LIMIT; x++) {
+			int p = sumOfAllFactors(x);
+			if(p <= LIMIT && x != p) {
+				if(x == sumOfAllFactors(p)) {
+					checked1.add(x);
+				}
+			}
+		}
+		f1 = System.currentTimeMillis();
+		System.out.println(checked1);
+		System.out.println("All Done");
+		System.out.println("The total time was " + (f1-s1) + " milliseconds");
+		
+		*/
+		
+		
+		//This is way faster than the one up top: finds the sum of the factors way faster
+		//Also, the numbers are in order from smallest to largest -> Some pairs are not next to each other
+		//Two systems
+		s2 = System.currentTimeMillis();
+		checked2 = new ArrayList<Integer>();
+		
+		for(int x = 2; x <= LIMIT; x++) {
+			int p = sumOfFactorsFastVersion(x);
+			if(p <= LIMIT && x != p) {
+				if(x == sumOfFactorsFastVersion(p)) {
+					checked2.add(x);
```

Comment:
put `p` in there as well
also you would want to check if `checked2.contains(x) || checked2.contains(x)` 
In this scenario, if you print every single loop out with sysout, it would appear like this:
220 284
284 220 
1184 1210
1210 1184
which doubles your time